src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -2119,7 +2119,7 @@ public void Kill_ExitedChildProcess_DoesNotThrow(bool killTree)
             Process process = CreateProcess();
             process.Start();
 
-            process.WaitForExit();
+            Assert.True(process.WaitForExit(Helpers.PassingTestTimeoutMilliseconds));
```

Comment:
consider adding string arg to the assert (i.e. `test timed out`) so that message doesn't look cryptic in the logs if this fails

Reply:
updated.